src/org/usfirst/frc/team2175/subsystem/ShooterSubsystem.java

```diff
@@ -71,7 +68,8 @@ public ShooterSubsystem() {
     }
 
     public void setShooterDefaultSpeed() {
-        shooterMotor.set(shooterSpeed);
+        shooterMotor.set(-shooterSpeed);
+        shooterMotorTwo.set(shooterSpeed);
```

Comment:
You can do if using a method in the CAN Talon library, and I think that you can do it at talon firmware level, also.